benchmarks/asv.conf.json

```diff
@@ -29,7 +29,7 @@
     "matrix": {
         "numpy": [],
         "Tempita": [],
-        "Cython": ["0.29.13"],
+        "Cython": ["0.29.14"],
```

Comment:
Please revert changes to benchmark files.

Reply:
I did not notice the file was in the commit. My benchmarks need Cython 0.29.14. I'll revert & keep my change local.